screens/home/HomeFeed.tsx

```diff
@@ -207,6 +215,7 @@ const HomeFeed = () => {
           }
         />
       }
+      ListFooterComponent={renderSpinner}
       onEndReached={getNextPosts}
```

Comment:
is there any kind of weird dead zone in which the spinner renders but onEndReached doesn't trigger? If not, lgtm